javascript/node/selenium-webdriver/BUILD.bazel

```diff
@@ -93,39 +93,115 @@ genrule(
 )
 
 nodejs_binary(
-    name = "cdp-srcs-generator",
+    name = "cdp-srcs-generator-v84",
     data = [
-        ":browser_protocol",
-        ":js_protocol",
+        ":browser_protocol_v84",
+        ":js_protocol_v84",
     ],
     entry_point = "devtools/generator/protocol-dts-generator.js",
 )
 
+nodejs_binary(
+    name = "cdp-srcs-generator-v85",
+    data = [
+        ":browser_protocol_v85",
+        ":js_protocol_v85",
+    ],
+    entry_point = "devtools/generator/protocol-dts-generator.js",
+)
+
+nodejs_binary(
+    name = "cdp-srcs-generator-v86",
+    data = [
+        ":browser_protocol_v86",
+        ":js_protocol_v86",
+    ],
+    entry_point = "devtools/generator/protocol-dts-generator.js",
+)
+
+copy_file(
```

Comment:
There's going to be plenty of these. You may want to use a list comprehension to generate the targets.